### PR TITLE
Logout button

### PR DESCRIPTION
**Pull request by BKSalman**
this was a pain to figure out for some fucking reason

### CODE REVIEW

**Comment by mohad12211** [comment]
is there a reason that you made logout a page instead of calling the endpoint in a callback of the button click?

**Comment by BKSalman** [comment]
Bruh, you're right, no reason honestly, but haven't tested it, but it could be useful if doing a logout after some logic or something

Basically can be reused is what I'm trying to say

**Review by mohad12211 — APPROVED** [review]
While I don't like that it's page, because the user can be redirected to it somehow involuntarily, as opposed to a voluntary action like clicking a button, it's not an issue for now.

**Comment by BKSalman** [comment]
Fair point, will try the other method later ™️

**Comment by BKSalman** [comment]
removed all useless shit, and replaced it with a button in the sidebar

File: client/src/lib/components/Sidebar.svelte [hunk 1]
```diff
@@ -1,29 +1,47 @@
...
+                            >New comic</Text>
+                    {/if}
+                </a>
+            </li>
+            <li>
+                <button class="link" on:click={logout}>
+                    <Fa size="2.5x" icon={faPersonWalking} />
+                    {#if open}
+                        <Text fontSize="xl" --margin="0 0 0 1rem"
+                            >New comic</Text>
```
Comment (mohad12211) [inline client/src/lib/components/Sidebar.svelte]: ^C^V

Comment (BKSalman) [reply client/src/lib/components/Sidebar.svelte]: Classic

Comment (mohad12211) [inline client/src/lib/components/Sidebar.svelte]:
```suggestion
                            >Logout</Text>
```

**Pull request merged**